training-react/login/src/index.js

```diff
@@ -0,0 +1,16 @@
+import App from '@components/App';
```

Comment:
I think that you are confused, the `Login` component should be together with `Game` screen, not in another folder